### PR TITLE
Fix Makefile, fix changelog, manifest, docs, tools.go

### PR DESCRIPTION
**Pull request by micbar**

### CODE REVIEW

**Review by refs — APPROVED** [review]

**Comment by refs** [comment]
@micbar consider adding a .github folder? Currently only the owner of the PR can merge it? I cannot merge it 🤷‍♂

File: template/changelog/CHANGELOG.tmpl [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {{ if gt (len $allVersions) 1 -}}
 # Changelog for [{{ .Version }}] ({{ .Date }})
 
-The following sections list the changes in ownCloud`}}{{ Name }}{{`{{ .Version }}.
+The following sections list the changes in `}}{{ Name }} {{`{{ .Version }}.
```
Comment (tboerger) [inline template/changelog/CHANGELOG.tmpl:6]: This looks wrong

File: template/docker/manifest.tmpl [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+{{`image: owncloud/`}}{{ Name }}{{`:{{#if build.tag}}{{trimPrefix "v" build.tag}}{{else}}latest{{/if}}
 {{#if build.tags}}
 tags:
 {{#each build.tags}}
   - {{this}}
 {{/each}}
 {{/if}}
 manifests:
-  - image: owncloud/`}}{{ Name }} {{`:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}linux-amd64
+  - image: owncloud/`}}{{ Name }}{{`:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}linux-amd64
```
Comment (tboerger) [inline template/docker/manifest.tmpl:9]: Same here

File: template/docker/manifest.tmpl [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
     platform:
       architecture: amd64
       os: linux
-  - image: owncloud/`}}{{ Name }} {{`:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}linux-arm64
+  - image: owncloud/`}}{{ Name }}{{`:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}linux-arm64
```
Comment (tboerger) [inline template/docker/manifest.tmpl:13]: Same here

File: template/docker/manifest.tmpl [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
     platform:
       architecture: arm64
       variant: v8
       os: linux
-  - image: owncloud/`}}{{ Name }} {{`:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}linux-arm
+  - image: owncloud/`}}{{ Name }}{{`:{{#if build.tag}}{{trimPrefix "v" build.tag}}-{{/if}}linux-arm
```
Comment (tboerger) [inline template/docker/manifest.tmpl:18]: Same here

File: template/docs/building.md [hunk 5]
```diff
@@ -12,11 +12,11 @@ geekdocFilePath: building.md
 As this project is built with Go, so you need to install that first. The installation of Go is out of the scope of this document, please follow the official documentation for [Go](https://golang.org/doc/install), to build this project you have to install Go >= v1.12. After the installation of the required tools you need to get the sources:
 
 {{`{{< highlight txt >}}
-git clone https://github.com/owncloud/{{ Name }}.git
-cd {{ Name }}
+git clone https://github.com/owncloud/`}}{{ Name }}{{`.git
```
Comment (tboerger) [inline template/docs/building.md:15]: Same here

File: template/docs/getting-started.md [hunk 6]
```diff
@@ -159,16 +159,16 @@ The program provides a few sub-commands on execution. The available configuratio
...
 {{< / highlight >}}`}}
 
 ### Health
 
 The health command is used to execute a health check, if the exit code equals zero the service should be up and running, if the exist code is greater than zero the service is not in a healthy state. Generally this command is used within our Docker containers, it could also be used within Kubernetes.
 
-{{`{{< highlight txt >}}
-{{ Name }} health --help
+{{`{{< highlight txt >}}`}}
+{{ Name }}{{` health --help
```
Comment (tboerger) [inline template/docs/getting-started.md:171]: Same here

File: template/docs/building.md (hunk 5 above)
Comment (tboerger) [reply template/docs/building.md:15]: Scrath this one

File: template/changelog/CHANGELOG.tmpl (hunk 1 above)
Comment (micbar) [reply template/changelog/CHANGELOG.tmpl:6]: Can you specify how wrong?
go Variables that should not be parsed by boilr needs to be escaped.

Comment (tboerger) [reply template/changelog/CHANGELOG.tmpl:6]: But escaping by an closing tag?

Comment (micbar) [reply template/changelog/CHANGELOG.tmpl:6]: looks, wrong, but the opening tag is there

File: template/docker/manifest.tmpl (hunk 3 above)
Comment (micbar) [reply template/docker/manifest.tmpl:13]: same

File: template/docs/building.md (hunk 5 above)
Comment (micbar) [reply template/docs/building.md:15]: same

File: template/docs/getting-started.md (hunk 6 above)
Comment (micbar) [reply template/docs/getting-started.md:171]: same

File: template/docker/manifest.tmpl (hunk 4 above)
Comment (micbar) [reply template/docker/manifest.tmpl:18]: same

**Pull request merged**